cnf-tests/.konflux/redhat.repo

```diff
@@ -0,0 +1,27 @@
+[rhel-9-for-$basearch-appstream-rpms]
+name = Red Hat Enterprise Linux 9 for $basearch - AppStream (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/rhel9/$releasever/$basearch/appstream/os
```

Comment:
`$releasever` will resolve to `9` which means the latest UBI 9 version (UBI 9.5). You will have a mix of 9.4 RPMs from the base image and 9.5 RPMs from the RPMs you want to additionally install. You can see those 9.5 RPMs in the `rpms.lock.yaml` file.

Reply:
good comment and I agree. I'm addressing this in #2285  and waiting for a recommended way to pass the full x.y stream version while manually generating this and during the prefetch step by the integration test.

Reply:
https://github.com/openshift-kni/cnf-features-deploy/pull/2285/commits/005a9d5ac75a777fe5a786931f604233fa868c73#diff-820b98e7bc57b3ec683c00a0fb7e167c32796d66242b25c2ace5d30427bbb222R17